sw/airborne/modules/display/max7456.c

```diff
@@ -396,7 +400,7 @@ static bool _osd_sprintf(char *buffer, char *string, float value)
   if (frac_nb > 3) {
     frac_nb = 3;       // Bound value.
   }
-  y = (sizeof(to_asc) - 1); // Point y to the end of the array.
+  y = (sizeof(to_asc) - 2); // Point y to the end of the array.
```

Comment:
not sure about this here... now it doesn't point to the last element of the array anymore...
if this is intended, the comment should be updated...